src/context.ts

```diff
@@ -160,11 +117,7 @@ export class Context {
     return (this.callbackQuery ?? this.chosenInlineResult)?.inline_message_id
   }
 
-  get passportData() {
-    if (this.message == null) return undefined
-    if (!('passport_data' in this.message)) return undefined
-    return this.message?.passport_data
-  }
```

Comment:
Redundant expressions to check if `message` exists in the object and there is `passport_data` in the message:
`if (this.message == null) return undefined`
`if (!('passport_data' in this.message)) return undefined`
We can use [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) here:
`return this?.message?.passport_data`

> The ?. operator functions similarly to the . chaining operator, except that instead of causing an error if a reference is nullish (null or undefined), the expression short-circuits with a return value of undefined

So this will return undefined if there is no `passport_data` defined or no `message` at all